Extensions/notificationblock.js

```diff
@@ -21,311 +21,173 @@ XKit.extensions.notificationblock = new Object({
-						$(n_box).find(".xkit-notification-notification-block-button").css("right", (35 + $(n_box).find(".xkit-reply-button").width() + 5) + "px");
-					}
-
-					if ($(n_box).hasClass("xkit-old-notifications")) {
-						$(n_box).find(".xkit-notification-notification-block-button").css("top", "16px");
-					} else {
-						$(n_box).find(".xkit-notification-notification-block-button").css("top", "12px");
-					}
-					if ($(n_box).hasClass("stretchy_kid_container") === true) {
-						if ($(n_box).find(".xkit-reply-button").length > 0) {
-							console.log("Has One-Click Reply button, pushing! 1");
-							$(n_box).find(".xkit-notification-notification-block-button").css("right", (8 + $(n_box).find(".xkit-reply-button").width() + 5) + "px");
-						} else {
-							$(n_box).find(".xkit-notification-notification-block-button").css("right", "8px");
-						}
-					} else {
-						if ($(n_box).hasClass("xkit-old-notifications")) {
-
-							if ($(n_box).find(".xkit-reply-button").length > 0) {
-								console.log("Has One-Click Reply button, pushing! 2");
-								$(n_box).find(".xkit-notification-notification-block-button").css("right", (43 + $(n_box).find(".xkit-reply-button").width() + 5) + "px");
-							} else {
-								$(n_box).find(".xkit-notification-notification-block-button").css("right", "43px");
-							}
-						} else {
-							if ($(n_box).find(".xkit-reply-button").length > 0) {
-								console.log("Has One-Click Reply button, pushing! 3");
-								$(n_box).find(".xkit-notification-notification-block-button").css("right", (38 + $(n_box).find(".xkit-reply-button").width() + 5) + "px");
-							} else {
-								$(n_box).find(".xkit-notification-notification-block-button").css("right", "38px");
-							}
-						}
-					}
-				}
-				var m_right = 45 + $(n_box).find(".xkit-notification-notification-block-button").width();
-				if ($(n_box).find(".xkit-reply-button").length > 0) {
-					console.log("Has One-Click Reply button, pushing! 15");
-					m_right += $(n_box).find(".xkit-reply-button").width() + 8;
-				}
-				if (XKit.extensions.notificationblock.added_css !== true) {
-					XKit.tools.add_css(".notification .block, .notification .ignore { right: " + m_right + "px !important; }", "notificationblock_notfix");
-					XKit.extensions.notificationblock.added_css = true;
-				}
-
-			}, 400);
-
-		}
+		$(notificationClasses.map(cls => `.${cls}.xnotificationblockchecked`).join(", ")).
+			removeClass("xnotificationblockchecked").removeAttr('data-xkit-notificationblock-blocked');
 
+		this.do();
 	},
 
 	unblock: function(post_id) {
-
-		var m_index = XKit.extensions.notificationblock.blacklisted.indexOf(post_id);
+		var m_index = this.blacklisted.indexOf(post_id);
 		if (m_index !== -1) {
-			XKit.extensions.notificationblock.blacklisted.splice(m_index, 1);
-			XKit.storage.set("notificationblock", "posts", XKit.extensions.notificationblock.blacklisted.join(","));
+		 this.blacklisted.splice(m_index, 1);
```

Comment:
nit: indentation

Reply:
to the [eslint], this is normal indentation.